distribution/lib/Standard/Base/0.2.27-SNAPSHOT/src/Data/Text/Extensions.enso

```diff
@@ -269,6 +565,33 @@ Text.utf_8 = Vector.Vector (Text_Utils.get_bytes this)
 Text.from_utf_8 : Vector.Vector Byte -> Text
 Text.from_utf_8 bytes = Text_Utils.from_utf_8 bytes.to_array
 
+## ADVANCED
+
+   Returns a vector containing the UTF-16 characters that encode the input text.
+
+   This is useful for low-level operations, such as binary data encoding and
+   decoding.
+
+   > Example
+     Get the UTF-16 bytes of the text "Hello".
+
+         "Hello".utf_16
+Text.utf_16 : Vector.Vector Char
```

Comment:
Should probably be `Vector.Vector Integer`. We don't support bare chars in our numbers state machine.